lib/tapjoy/ldap/cli/group/delete.rb

```diff
@@ -0,0 +1,36 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Delete LDAP group
+        class Delete
+          def delete
+            confirm unless opts[:force]
+            puts Tapjoy::LDAP::API::Group.destroy(opts[:name])
+          end
+
+          private
+          def opts
+            @opts ||= Trollop::options do
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not use `::` for method calls.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2633306022/issues/source?bid=3316970&fileBranchId=3318857#l14)